packages/DropdownMenu/src/Confirmation/Confirmation.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import ContentContainerStyled from "../ContentContainer.styles";
+import ConfirmationStyles from "./Confirmation.styles";
+
+const { func, string } = PropTypes;
+
+const propTypes = {
+  buttonSize: string,
```

Comment:
This should be constrained to `ShirtSizes`... but actually, to JP's point, I kind of think we should have a size prop for the whole `<DropdownMenu>` that will dictate the sizing of _everything_.  A little more complicated, but easier to use, and enforces consistency.

Reply:
Ok i've restricted it to ShirtSizes.Default